Aura Operating System/Aura OS/System/Users/Setup.cs

```diff
@@ -300,34 +312,41 @@ private void Step4()
             string user = text.Remove(middle, text.Length - middle);
             string pass = text.Remove(0, middle + 6);
 
-            if (File.Exists(@"0:\System\Users\" + user + ".usr"))
+            if (user == "root" || pass == "root")
```

Comment:
So root password cannot be change.
It's not a good idea.

Reply:
Done.